docs/porting/overview.md

```diff
@@ -1,5 +1,82 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+By [Phillip Carter](https://github.com/cartermp)
+
+If you've got code running on .NET Framework you may be interested in running your code on .NET Core.  This article covers evaluating if porting to .NET Core is reasonable for you, an overview of the porting process, and a list of the tools you may use to port.
+
+## Should you port?
+
+The first thing you need to do is evaluate if it makes sense to port .NET Framework code over to .NET Core in the first place.  .NET Core offers some attractive capabilities, such as:
+
+- Using ASP.NET Core to write great web apps and services which you can write and deploy on OS X, supported Linux distributions, or Windows
+- The ability to deploy apps and services with different versions of the .NET Core runtime installed side-by-side
+- The ability to write a UWP app that will run on multiple Windows 10 devices
+- The ability to utilize the advancements of .NET Core while your code lives seamlessly alongside .NET Framework code, such as a WinForms GUI consuming .NET Core microservices
+
+Your code could be a good candidate for porting to .NET Core if...
+
+- You have an ASP.NET web app that you'd like to deploy to more than .NET Framework
```

Comment:
I don't understand this sentence. What do you mean by deploy to more than .NET Framework?